### ISSUE TITLE
Usability issues

### ISSUE THREAD

**Issue by dbrgn** [isacikgoz/tldr#33]
Hi fellow tldr implementer! I wanted to try your client, so I installed 0.6.1 on Arch via AUR.

When I run it the first time, it looks like this:

![2020-09-04-001725_992x197_scrot](https://user-images.githubusercontent.com/105168/92178965-05c1df80-ee44-11ea-8b34-16f0bb0208a6.png)

(Note: This must have something to do with some control characters, since when I make the window small and large again, there is a proper line break. I'm using alacritty as a terminal emulator, in case that's relevant.)

So since the command asked me to re-run it with `-u`, I did so. However, this results in the same warning being printed again.

![2020-09-04-001845_999x244_scrot](https://user-images.githubusercontent.com/105168/92179049-3b66c880-ee44-11ea-9d4d-6c2141dbf6ec.png)

So I thought, maybe I need to combine `-u` with a page name. But that doesn't seem to do anything either...

![2020-09-04-002051_999x244_scrot](https://user-images.githubusercontent.com/105168/92179183-7f59cd80-ee44-11ea-85d3-0ec8d464d47c.png)

Another theory was that it's broken in the interactive mode, but works in static mode. But with `tldr -s -u`, the cache isn't pulled either.

TL;DR: I don't know how to use your tool :smile: It could well be that I'm too stupid to use it properly, but my experience may be useful to improve the API!
